tests/test_models.py

```diff
@@ -0,0 +1,173 @@
+from datetime import datetime, timedelta
+
+from django.conf import settings
+
+import pytest
+
+from .factories import (ValueFactory, ValuesetFactory, MetadataFactory,
```

Comment:
This is an _import smell_ to me :stuck_out_tongue_closed_eyes: 

Probably better of with 

```python 
from . import factories
```